options/options_helper.cc

```diff
@@ -255,13 +254,10 @@ std::unordered_map<std::string, CompressionType>
         {"kDisableCompressionOption", kDisableCompressionOption}};
 #ifndef ROCKSDB_LITE
 
-const std::string kNameComparator = "comparator";
 const std::string kNameEnv = "env";
-const std::string kNameMergeOperator = "merge_operator";
 
-template <typename T>
 Status GetStringFromStruct(
```

Comment:
If I understand correctly, in the current implementation, GetStringFromStruct is only used in options_helper.cc as a template function. In this PR, this function is also used in other places. Can we remove the forward declaration and include options_helper.h instead?